src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -64,6 +64,15 @@ internal ConsoleHostUserInterface(ConsoleHost parent)
 #if UNIX
             SupportsVirtualTerminal = true;
 #else
+            SupportsVirtualTerminal = TryTurnOnVtMode();
+#endif
```

Comment:
We could move this in `TryTurnOnVtMode()` and have here:
```suggestion
            SupportsVirtualTerminal = TryTurnOnVtMode();
```

Reply:
Since we compile specifically for non-Windows, seems unnecessary to have that helper for non-Windows.

Reply:
`#if UNIX` would be in TryTurnOnVtMode(). This makes the code more readable.